packages/component-library-vue/src/UtrechtSelect.vue

```diff
@@ -23,10 +34,18 @@
     :disabled="disabled"
     :readonly="readonly"
     :required="required"
-    :value="modelValue"
-    @input="$emit('update:modelValue', (($event as InputEvent).target as HTMLSelectElement).value)"
   >
-    <slot />
+    <option
+      v-for="{value, label} in options"
+      :key="value"
+      :value="value"
+      :class="[
+        {'utrecht-select__option--disabled': disabled},
```

Comment:
The `disabled` attribute and the `utrecht-select__option--disabled` class name should only be used when this specific option is disabled, not when the entire select is disabled.